### ISSUE TITLE
Observatory timeline is no longer working

### ISSUE THREAD

**Issue by liyuqian** [flutter/flutter#50833]
Clicking "refresh" button in Observatory timeline has no effect at all. Flutter version 48d5fcabbd805a622b74af2d2671ff0c845abdcd

However, DevTools can get the timeline events with the Observatory URL.

**Comment by aam** [comment]
cc @rmacnak-google

**Comment by liyuqian** [comment]
cc @a-siva @bkonyi 

I think @flar has experienced similar issues.

**Comment by aam** [comment]
I believe this is due to WebComponents V0 support being dropped from most version of Chrome.

**Comment by bkonyi** [comment]
This should be fixed as of [this CL](https://dart.googlesource.com/sdk/+/82e2d4cabcdf4076ca27e22e3d277c09dc8903cb). This is likely a temporary workaround as Catapult will stop working eventually, but it should be enough for now.

Until the CL above rolls into Flutter, you can always save your timeline and then import it to the viewer in `about://tracing`.

**Comment by jason-simmons** [comment]
See https://bugs.chromium.org/p/chromium/issues/detail?id=1036492

You can work around this by viewing the timeline with a build of Chrome before version 80

**Comment by liyuqian** [comment]
Thank you for the quick responses! A relative note: I exported a tracing with DevTools yesterday
[timeline_2020_2_13-1581662382688000.json.zip](https://github.com/flutter/flutter/files/4207133/timeline_2020_2_13-1581662382688000.json.zip)
and it can't be opened by `about:tracing`, but it can be opened by perfetto.

@kenzieschmoll @devoncarew : do you plan to integrate DevTools with perfetto? Perfetto seems to be quite fast, and preserves some nice keyboard shortcuts from `about:tracing` (Catapult?).

**Comment by kenzieschmoll** [comment]
I just tried and am able to import a DevTools json file into both chrome://tracing and perfetto. The trace you linked may be hitting an edge case.

We haven't looked into replacing our trace viewer with Perfetto or embedding it (if that's even possible). The DevTools trace viewer has some custom UI (like guidelines for enhanced async tracing) and usability features (like scroll to zoom in addition to WASD) that we've added and don't want to lose. We should however, ensure that DevTools traces can be imported into Perfetto (which seems like is already possible).

What are the keyboard shortcuts you are referring to?
